src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1818,17 +1802,10 @@ let internal_commands logger =
           Parallel.init_master () ;
           let logger = Logger.create () in
           let conf_dir = Mina_lib.Conf_dir.compute_conf_dir conf_dir in
-          let genesis_constants =
-            Genesis_constants.Compiled.genesis_constants
-          in
-          let constraint_constants =
-            Genesis_constants.Compiled.constraint_constants
-          in
-          let proof_level = Genesis_constants.Proof_level.Full in
           let%bind precomputed_values, _ =
             Genesis_ledger_helper.Config_loader.load_config_files ~logger
-              ~conf_dir ~genesis_dir ~genesis_constants ~constraint_constants
-              ~proof_level config_files ~cli_proof_level:None
+              ~conf_dir ?genesis_dir ~cli_proof_level:Full ~itn_features
```

Comment:
`cli_proof_level` is used here so that whatever is provided in compile-time config and initialized via `load_config_files` is overridden with `Full`.